### PR TITLE
Updates to theory manual and add examples.

### PR DESCRIPTION
**Pull request by aeslaughter**
@sveerara This rebases your work with the changes needed for changes to MooseDocs. It also fixes the images that were add with LFS.

### CODE REVIEW

**Comment by aeslaughter** [comment]
I am still working on this.

File: doc/content/manuals/user/index.md [hunk 1]
```diff
@@ -1,4 +1,126 @@
...
+
+### Materials
+#### Linear elastic soil
+A linear elastic soil can be defined in MASTODON by defining a elasticity tensor using ComputeIsotropicElasticityTensor, stress calculator using ComputeLinearElasticStress, and a small strain calculator using ComputeSmallStrain. Density can be defined using the GenericConstantMaterial. An example of these input blocks is shown below:
+
+!listing test/tests/dirackernels/seismic_source/one_seismic_source.i start=Materials end=Executioner
+
+#### I-soil material model
+The nonlinear soil material model usage is demonstrated in the [Example 1] (manuals/examples/index.md)
+>>>>>>> Changes to theory manual, additions to user and examples manuals #3
```
Comment (sveerara) [inline doc/content/manuals/user/index.md]: This line is probably left over from the rebase.

File: doc/content/manuals/theory/index.md [hunk 2]
```diff
@@ -1,21 +1,49 @@
...
+
+This document describes the theoretical and numerical foundations of
+MASTODON.
 
 ## Governing equations
 The basic equation that MASTODON solves is the nonlinear wave equation:
 
 \begin{equation}
 \label{eqn:governing_equation}
+<<<<<<< de2079b461234601978c3cd2e70cb5c6fbcfd7f3
```
Comment (sveerara) [inline doc/content/manuals/theory/index.md]: Needs to be removed

File: doc/content/manuals/theory/index.md [hunk 3]
```diff
@@ -1,21 +1,49 @@
...
+\end{eqnarray}
+
+where $\rho$ is the density of the soil or structure that can vary with space, $\boldsymbol{\sigma}$ is the stress at any point in space and time, $\mathbf{F_{ext}}$ is the external force acting on the system that can be in the form of localized seismic sources or global body forces such as gravity, and $\mathbf{\ddot{u}}$ is the acceleration at any point within the soil-structure domain. The left side of the equation contains the internal forces acting on the system with first term being the contribution from the inertia, and the second term being the contribution from the stiffness of the system. Additional terms would be added to this equation when damping is present in the system. The material stress response ($\boldsymbol{\sigma}$) is described by the constitutive model, where the stress is determined as a function of the strain ($\boldsymbol{\epsilon}$), i.e. $\boldsymbol{\sigma}(\boldsymbol{\epsilon})$. Details about the material constitutive models available in MASTODON are presented in the section about [Material models](##Material models).
+>>>>>>> Changes to theory manual, additions to user and examples manuals #3
```
Comment (sveerara) [inline doc/content/manuals/theory/index.md]: The earlier paragraph from commit de2079b461234601978c3cd2e70cb5c6fbcfd7f3 should be kept and the above paragraph should be removed.

File: doc/content/manuals/theory/index.md [hunk 4]
```diff
@@ -167,39 +242,84 @@ The I-soil material model is a nonlinear hysteretic soil model that is based on
...
+The current version of I-soil implemented in MASTODON utilizes Masing type un/reloading behavior and is analogous to MAT79 (LS-DYNA) material model but does not exhibit numerical instability observed in MAT79 (\citet{numanoglu2017conf}). Masing type un/reloading is inherently achieved by the model because upon un/reloading the yielded nested components regain stiffness and strength. The cyclic response obtained from current version of the model is presented in \ref{fig:1D_isoil_representation}. Reduction factor type modification on un/reloading behavior (\citet{phillips2009damping}; \citet{numanoglu2017nonmasing}) is an ongoing study within MASTODON framework.
+>>>>>>> Changes to theory manual, additions to user and examples manuals #3
```
Comment (sveerara) [inline doc/content/manuals/theory/index.md]: This newer version should be kept and the older version from de2079b461234601978c3cd2e70cb5c6fbcfd7f3 should be removed.

File: doc/content/manuals/theory/index.md [hunk 5]
```diff
@@ -167,39 +242,84 @@ The I-soil material model is a nonlinear hysteretic soil model that is based on
...
+  mobilized by the soil at large strains. The parameters theta\_1 through 5 are the curve
+  fitting parameters and can be obtained using DEEPSOIL
+  \citep{hashash2016deepsoil}. Other than the number of points, all the
+  other parameters can be given as a vector for the different soil
+  layers. The number of points, which determines the number of
+  elastic-perfectly plastic curves to be generated, is constant for all soil layers.
+
+Once the backbone curve is provided to I-soil, the model determines
+the properties for nested components presented in \ref{fig:1D_isoil_representation}. The stress integration for each nested component follows  classic elastic predictor - plastic corrector type radial return algorithm (\citet{simo2006computein}) and model stress is obtained by summing the stresses  from each nested component:
+>>>>>>> Changes to theory manual, additions to user and examples manuals #3
```
Comment (sveerara) [inline doc/content/manuals/theory/index.md]: This newer changes should be retained.

File: doc/content/manuals/user/index.md [hunk 6]
```diff
@@ -1,4 +1,126 @@
...
+
+### Materials
+#### Linear elastic soil
+A linear elastic soil can be defined in MASTODON by defining a elasticity tensor using ComputeIsotropicElasticityTensor, stress calculator using ComputeLinearElasticStress, and a small strain calculator using ComputeSmallStrain. Density can be defined using the GenericConstantMaterial. An example of these input blocks is shown below:
+
+!listing test/tests/dirackernels/seismic_source/one_seismic_source.i start=Materials end=Executioner
+
+#### I-soil material model
+The nonlinear soil material model usage is demonstrated in the [Example 1] (/examples/index.md)
+>>>>>>> Changes to theory manual, additions to user and examples manuals #3
```
Comment (sveerara) [inline doc/content/manuals/user/index.md]: @aeslaughter, Please remove the above line from the user manual.

**Comment by sveerara** [comment]
Thanks for correcting the theory manual. There is just one line on the user manual that needs to be removed. Once that is done, I will merge the request. Thanks again for your help with updating the manuals and making sure the conversion to the website works correctly.

File: doc/content/manuals/user/index.md (hunk 6 above)
Comment (aeslaughter) [reply doc/content/manuals/user/index.md]: Dang, thanks for catching this. My search failed me.

**Pull request merged**